tla/Traceccfraft.tla

```diff
@@ -113,7 +113,7 @@ OneMoreMessage(msg) ==
 VARIABLE l, ts
 
 TraceInit ==
-    /\ l = 1
+    /\ l = 2
```

Comment:
why 2 now?

Reply:
The first line of the log maps to the initial state.  The second line (`l=2`) maps to the second state.  In other words, there is always a stuttering step after the initial state with `l=1`.